compose/material3/material3/src/jsWasmMain/kotlin/androidx/compose/material3/PlatformDateFormat.jsWasm.kt

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.material3
+
+import androidx.compose.ui.text.intl.Locale
+import kotlin.js.Date
+import kotlinx.datetime.Instant
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.atStartOfDayIn
+import kotlinx.datetime.toLocalDateTime
+
+internal actual object PlatformDateFormat {
+
+    actual val weekdayNames: List<Pair<String, String>>?
+        get() = weekdayNames()
+
+    actual val firstDayOfWeek: Int
+        get() = firstDayOfWeek()
+
+    private const val NUMERIC = "numeric"
+    private const val SHORT = "short"
+    private const val LONG = "long"
+
+    private val countriesFirstDow: Map<String, Int> by lazy {
```

Comment:
Could you please explain what this map means? I can't get it from the val name and from the content.
Is it expected that some values are duplicated? I noticed: US, IN, KE, AF, etc. 

I see it's used in `private fun firstDayOfWeek()`
`return countriesFirstDow[Locale.current.region.uppercase()] ?: 1`

if the tags represent region codes, why are they not unique?
Is it expected to be the map regions where the first day of the week is Saturday and Sunday?

Reply:
Removed duplicates, renamed val